### DISCUSSION TITLE
Gopro 11 sharpness low - medium

### DISCUSSION THREAD

**Discussion by Pax709** [gopro/labs#508] [Q&A]
I wonder what the sharpening setting to low (ProTune GP11) does , when sharpened in davinci resolve you see more detail and grain compared to the medium and high settings. What is the result of this, does low sharpening affect the reduction of denoise?

**Comment by Pax709** [comment]
![LOW_VS_MEDIUM](https://github.com/gopro/labs/assets/141012631/581c52f2-d234-4555-a5c3-7de64a63d1ff)
![Zrzut ekranu (261)](https://github.com/gopro/labs/assets/141012631/339b9ced-7577-440f-bbe3-9b1c752004a5)

**Reply by Pax709** [reply]
I would like to know the opinion whether it is worth playing with low sharpening, you can see that on low there is more detail, but it seems that the image is inferior in quality ? My guess is that Gopro uses something more than a simple sharpening filter in Davinci Resolve.

**Reply by dnewman-gpsw** [reply]
I'm sure the tuning is different for each mode.  With higher sharpening it looks like the denoise is tuned higher.  You could experiment with oMNR01=1 to have the denoise similar on all modes.

**Reply by Pax709** [reply]
I think that viewing only at 100% magnification is better low sharpening ;), high magnifications are not suitable here.

**Reply by kouyoumin** [reply]
Does NR01 set a absolute denoise strength or something relative to default?
NR01=100 is supposed to be equal to default.
But sharpness setting seems to affect the default?

**Reply by dnewman-gpsw** [reply]
Not sure why. NR01=100 is the same disabling the control at NR01=0, sharpen is not controlled by this feature.
